bin/keymanager/KeyManager.ts

```diff
@@ -1,6 +1,6 @@
 import commander from 'commander'
-import { actionRunner, pkLogger, PKMessageType } from '../polykey'
 import initPolyKey from '../initPolykey'
+import { actionRunner, pkLogger, PKMessageType } from '../polykey'
```

Comment:
Use `@polykey`?

Reply:
I got stuck here, I'm not sure how to use the module level imports for this one, at the moment `@polykey` resolves to the `src` directory. Perhaps we could have another `tsconfig path` for the bin folder? something like `@polykey-cli` perhaps?

Reply:
The cli source files have been moved to `src` directory so this is no longer an issue.
The polykey library is contained within `src/lib/` and has a path of `@polykey/...`.
The polykey cli is contained within `src/cli/` and has a path of `@polykey-cli/...`